### ISSUE TITLE
Setting camera direction in Android app not working

### ISSUE THREAD

**Issue by HolgerJeromin** [mapillary/mapillary_issues#2566]
### Basic information
* Platform:  `Android` 
* Version: `Beta from april 1st` 
* System/Browser: `Fairphone`

### Steps to reproduce behavior
Set angle to slight right while driving

### Expected behavior
All images should be 45 deg to the right

### Actual behavior
All images are 45 deg to the left.

https://www.mapillary.com/app/?lat=51.881230000000016&lng=8.55860800000005&z=17&pKey=wGBWLN6Looaf2kVdNmae4Q&focus=map

**Comment by jfburke3** [comment]
Is this related to the direction problem I reported in #2579 ?

**Comment by fgouget** [comment]
This issue does look like it's the same as the one I reported in 2746.

However the title is wrong: Mapillary did not systematically point the pictures 45 degrees to the left. Initially it points them 135 degrees to the left. Then after the first roundabout it changes that to 45 degrees to the left. Then later forward (i.e. roughly keeping the same compass orientation). And after a big curve 135 degrees to the right and then again straight ahead. In other words Mapillary keeps changing the pictures orientation relative to the car and to the geographic north (even when driving in a straight line in the latter case).

This is also what I observed in issue 2746.
Pictures started at 135 degrees left, and kept pointing south-east as the car went through the first loop, so that they ended up pointing 45 degrees left. Then they started pointing east so full ahead then 90 degrees left as the car turned, and finally they pointed north-east, that is 45 degrees left relative to the car.

Issue 2579 may be the same but some sequences I've seen have been normalized since making it hard to tell. The sequence posted on August 7 still clearly exhibits the issue (and also lots of GPS noise) although the poster seems to imply things are better.
